Makefile

```diff
@@ -20,9 +20,9 @@ gen:
 	etc/templates/... \
 	public/...
 
-test:
+test: build install
```

Comment:
why? Install for test doesn't seem right.

Reply:
done